render-ws-java-client/src/test/java/org/janelia/render/client/TilePairClientTest.java

```diff
@@ -102,11 +105,13 @@ public void testCrossDeriveWithMissingLayers() throws Exception {
         client.deriveAndSaveSortedNeighborPairs();
 
         final List<Path> pairFilePaths = new ArrayList<>();
-        Files.list(Paths.get(".")).forEach(path -> {
-            if (path.getFileName().toString().startsWith(baseFileName)) {
-                pairFilePaths.add(path);
-            }
-        });
+        try (final Stream<Path> files = Files.list(Paths.get("."))) {
+            files.forEach(path -> {
+                if (path.getFileName().toString().startsWith(baseFileName)) {
+                    pairFilePaths.add(path);
```

Comment:
It feels like a `.filter()` followed by a `.forEarch()` would be more idiomatic when using streams, here.

Reply:
good point